modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
+
+If conflicting modes are set by the `ALTER DATABASE` command and the configuration parameters, i.e. one says read-write and the other read-only, the database will be read-only and prevent write queries.
+
+[NOTE]
+====
+Modifying access mode is only available to standard databases and not composite databases.
+====
+
+
+=== Alter database access mode to read-only
+
+.Query
+[source, cypher]
+----
+ALTER DATABASE customers SET ACCESS READ ONLY
+----
+
+The database access mode can be seen in the `access` output column of the command `SHOW DATABASES`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES yield name, access
+----
+
+.Result
+[role="queryresult"]
+----
++----------------------------+
+| name        | access       |
++----------------------------+
+| "customers" | "read-only"  |
+| "movies"    | "read-write" |
+| "neo4j"     | "read-write" |
+| "system"    | "read-write" |
++----------------------------+
+----
+
+=== Alter database access mode to read-write
+
+`ALTER DATABASE` commands are optionally idempotent, with the default behavior to fail with an error if the database does not exist.
+Appending `IF EXISTS` to the command ensures that no error is returned and nothing happens should the database not exist.
+
+.Query
+[source, cypher]
+----
+ALTER DATABASE nonExisting IF EXISTS
+SET ACCESS READ WRITE
+----
+
+[role=enterprise-edition not-on-aura]
+[[manage-databases-stop]]
+== Stop databases
+
+Databases can be stopped using the command `STOP DATABASE`.
+
+.Query
+[source, cypher]
+----
+STOP DATABASE customers
+----
+
+The status of the stopped database can be seen using the command `SHOW DATABASE name`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASE customers YIELD name, requestedStatus, currentStatus
+----
+
+.Result
+[role="queryresult"]
+----
++-----------------------------------------------+
+| name        | requestedStatus | currentStatus |
++-----------------------------------------------+
+| "customers" | "offline"       | "offline"     |
++-----------------------------------------------+
+----
+
+[NOTE]
+====
+Databases that are stopped with the `STOP` command are completely shut down and may be started again through the `START` command.
+In a cluster, as long as a database is in a shutdown state, it can not be considered available to other members of the cluster.
+It is not possible to do online backups against shutdown databases and they need to be taken into special consideration during disaster recovery, as they do not have a running Raft machine while shutdown.
+Dropped databases are completely removed and are not intended to be used again at all.
+====
+
+[role=enterprise-edition not-on-aura]
+[[manage-databases-start]]
+== Start databases
+
+Databases can be started using the command `START DATABASE`.
+
+.Query
+[source, cypher]
+----
+START DATABASE customers
+----
+
+[NOTE]
+====
+Both standard databases and composite databases can be stopped using this command.
+====
```

Comment:
this note is missing from the stop database section above and says `stopped` here in the start database section